src/main/webapp/projectReview/ProjectReview.js

```diff
@@ -458,9 +469,9 @@ class ProjectReview extends Component {
       prev.formData.projectExtraProps[field] = value;
       return prev;
     },
-    () => {
-      if(this.state.errorSubmit == true) this.isValid() 
-    }); 
+      () => {
+        if (this.state.errorSubmit == true) this.isValid()
```

Comment:
Can't this be simplified ? (here and elsewhere)
```suggestion
        if (this.state.errorSubmit) this.isValid()
```